[Area 1 Email Security] New doc tile (#5125)
cloudflare/cloudflare-docs: content/email-security/email-configuration/lists/allowed-patterns.md

---
title: Allowed patterns
pcx-content-type: concept
weight: 1
---

# Allowed patterns

> When you set up **allowed patterns**, our email security exempts messages that match certain patterns from normal detection scanning.

Reviewer: our = Cloudflare email security? or Area 1?

## Add an allowed pattern

To create a new allowed pattern:

1. Log in to the [Area 1 dashboard](https://horizon.area1security.com/).
2. Go to **Settings** (the gear icon).
3. On **Email Configuration**, go to **Allow List** > **Allowed Patterns**.
4. Click **+ New Pattern**.
5. Enter the pattern information:

    - **Allowed Pattern**: Enter one of the following types of pattern:

        - *Email addresses*, which must be a valid email.
        - *IP addresses*, which can only be IPv4. IPv6 and CIDR are invalid entries.
        - *Regular expressions*, which must be [valid](https://www.freeformatter.com/java-regex-tester.html) Java expressions.
    
    - **Allow Type**: Choose one or more of the following types:

        - *Trusted Sender*: Messages will bypass all [detections](/email-security/reference/dispositions-and-attributes/) and link following by Area 1. Typically, only applies to phishing simulations from vendors such as KnowB4.

>         - *Exempt Recipient*: Will exempt messages from all Area 1 [detections](/email-security/reference/dispositions-and-attributes/) intended for recipients matching this pattern (email address or regular expression only). Typically, this only applies to submission mailboxes for user reporting to Security.

Reviewer: is there a reason security is capitalized?

        - *Acceptable Sender*: Will exempt messages from the `Spam`, `Spoof`, and `Bulk` [dispositions](/email-security/reference/dispositions-and-attributes/#available-values) (but not `Malicious` or `Suspicious`). Commonly used for external domains and sources that send mail on behalf of your organization, such as marketing emails or internal tools.
    
    - **Notes**: Provide additional notes about the allowed pattern.

> 6. Choose whether to **Verify Sender**, which applies the the `Spoof` [disposition](/email-security/reference/dispositions-and-attributes/#available-values) for emails failing the SPF/DKIM/DMARC domain policy (only applicable for *Trusted Sender* and *Acceptable Sender*). 

Reviewer: ```suggestion
6. Choose whether to **Verify Sender**, which applies the `Spoof` [disposition](/email-security/reference/dispositions-and-attributes/#available-values) for emails failing the SPF/DKIM/DMARC domain policy (only applicable for *Trusted Sender* and *Acceptable Sender*). 
```

7. Click **Save**.

### CSV uploads

> You can also upload a CSV file of multiple allowed patterns, so long as the file is smaller than 150 KB, starts with a header row of all required values, and contains no additional fields.

Reviewer: ```suggestion
You can also upload a CSV file of multiple allowed patterns. The CSV file must be smaller than 150 KB, start with a header row of all required values, and contain no additional fields.
```
